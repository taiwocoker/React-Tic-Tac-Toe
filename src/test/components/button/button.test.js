import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Button from '../../../components/atoms/button';

it('renders correctly', () => {
    const tree = renderer
        .create(
                <MemoryRouter>
                    <Button title={'This Works'} />
                </MemoryRouter>
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
