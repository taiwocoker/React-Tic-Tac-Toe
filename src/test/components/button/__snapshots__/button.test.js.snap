// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="sc-bdfBQB kHFbFs"
>
  This Works
</button>
`;
